tests/secure_verify/CMakeLists.txt

```diff
@@ -77,12 +77,28 @@ if (BUILD_ENCLAVES)
   add_test(
     NAME tests/secure_verify_tdx_v5_openssl
     COMMAND $<TARGET_FILE:secure_verify> secure_verify_enc_openssl.signed
-            ${CMAKE_CURRENT_SOURCE_DIR}/data/tdx_quote_v5 -f tdx_quote_v5
+            ${CMAKE_CURRENT_SOURCE_DIR}/data/tdx_quote_v5 -f tdx_quote
```

Comment:
what happened to `tdx_quote_v5` format?

Reply:
`tdx_quote_v5` was a typo. In code there's only `tdx_quote`
```
#define INPUT_PARAM_EVIDENCE_FORMAT_TDX_QUOTE "tdx_quote"
```

`tdx_quote_v5` happens to work because `strncmp` accepts any string with a `tdx_quote` prefix.